### DISCUSSION TITLE
printer.cfg x and y min and endstop

### DISCUSSION THREAD

**Discussion by MiknBit** [OpenNeptune3D/OpenNept4une#83] [Q&A] [answered]
First of all, thank you for the this amazing project! 
Recently, I experimented with the code on my Neptune 4 Pro, and I encountered an issue with the automatic mesh calibration feature. Specifically, when using the x and y min and endstop positions in the printer.cfg file of the OpenNeptune project, I received an error message stating "probe didn't trigger after full movement." This error occurred while the motors seemed to be struggling, almost as if they were attempting to move beyond their designated range.

However, when I substituted these values with the original Elegoo printer.cfg file, the issues went away. As my understanding, the measurements utilized in this project represent the distance between the nozzle and the edge of the print bed, with the axes at their minimum physical positions. These measurements appear to align with the actual dimensions of the printer.
Though the sensor is to the left side of the hotend, not aligned with the nozzle, maybe that could be the problem.

Am I missing anything, any thought on this?

**Comment by halfmanbear** [comment, marked as answer]
Yes those offsets tell the printer where the bed starts. These will be slightly different for each printer (manufacturing tolerance). 

There is also a probe offset section in the printer.cfg (this shouldn't ever deviate between models).

That being said I'm surprised the small xy endstop offset difference is enough to make your probe be off the edge of the bed causing a nozzle/bed crash.

What values are you currently using that work for you?

I assume you using the macro button within the fluidd interface to run the bed mesh? 

*Just to note If your doing it by manual command (BED_MESH_CALIBRATE) must preform a (G28) (full axis home) before doing it else the machine has no idea where the bed is.

**Reply by MiknBit** [reply]
Now that you make me think, it was later and I probably switched from using the macro to the manual command without noticing, and forgot a couple times about G28. 

The values I used (those different from those in OpenNept4une) were : 
position_min:  -10 (x) 
position_endstop: -8.0 (x)
position_endstop: 0.0 (y)

Didn't notice the presence of x and y offset of the probe, so now all the configs make sense! 
Reverted back to this project's values and works again!

I just can't figure out why Elegoo have measures wrong on their own product.
